src/parser/jobs/ast/modules/ArcanaUndrawUsage.tsx

```diff
@@ -0,0 +1,62 @@
+import {Trans, Plural} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+export default class ArcanaUndrawUsage extends Analyser {
+	static override handle = 'arcanaundraws'
+	static override dependencies = [
+		'suggestions',
+	]
```

Comment:
Remove this, in favor of the `@dependency private suggestions` import below - for TypeScript, we require using the `@dependency` decorator so that you get the correct typing on the imports when you reference them.